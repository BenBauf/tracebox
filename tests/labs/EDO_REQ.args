-p IP/TCP/EDOREQUEST/NOP/NOP
