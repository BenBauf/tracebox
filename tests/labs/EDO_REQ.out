tracebox to 1.2.3.4 (1.2.3.4): 64 hops max
1: 1.1.1.1 TCP::CheckSum -TCPOptionExtendedDataOffsetRequest 
2: 1.2.3.4 TCP::CheckSum IP::TTL IP::CheckSum -TCPOptionExtendedDataOffsetRequest 
